Docker/Runner/Environment.php

```diff
@@ -27,17 +27,27 @@ class Environment
      */
     private $forwardTokenDetails;
 
-    public function __construct(Client $storageClient, $configId, $forwardToken, $forwardTokenDetails)
+    /**
+     * @var array
+     */
+    private $configParameters;
+
+    public function __construct(Client $storageClient, $configId, array $component, array $configParameters)
     {
         $this->storageClient = $storageClient;
         $this->configId = $configId;
-        $this->forwardToken = $forwardToken;
-        $this->forwardTokenDetails = $forwardTokenDetails;
+        $this->forwardToken = $component['forward_token'];
+        $this->forwardTokenDetails = $component['forward_token_details'];
+        if ($component['inject_environment']) {
```

Comment:
tyjo, touhle kontrolou si nejsem úplně jistej. nedokážu říct proč, ale nějak mi to nesedí. parametry konstruktoru se vzájemně ovlivňují - spíš bych to vyhodil někam ven a $configParameters vůbec nepředával? jenže pak se to zase bude hůř testovat, že? :-/